modules/internal/ChapelError.chpl

```diff
@@ -368,6 +368,12 @@ module ChapelError {
     return chpl_fix_thrown_error(err.release());
   }
 
+  pragma "no doc"
```

Comment:
I'd probably mark this function with `pragma "last resort`".
Additionally I'd consider making different overloads for records and classes that don't inherit from error.

Reply:
Are you looking for a different overload to generate a different error message?

Reply:
It could. I also like to avoid making totally generic functions in the included-by-default code.